### PR TITLE
366 selecteditem binding is not working in list view

### PR DESCRIPTION
**Pull request by Sofiya-kumar**
If selected item is preselecting with dynamic binding its not showing as selected item in UI.

### CODE REVIEW

File: Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs [hunk 1]
```diff
@@ -29,87 +32,145 @@ public ICommand SelectionChangedCommand
...
         set => SetValue(SelectionModeProperty, value);
-        
-    }
-    public new IEnumerable ItemsSource
-    {
-        get => (IEnumerable)GetValue(ItemsSourceProperty);
-        set => SetValue(ItemsSourceProperty, value);
+
     }
+    //public new IEnumerable ItemsSource
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs:54]: Don't we need this?

File: Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs [hunk 2]
```diff
@@ -29,87 +32,145 @@ public ICommand SelectionChangedCommand
...
+    //    set => SetValue(ItemsSourceProperty, value);
+    //}
     #endregion
     #region Constructor
     public TMListView()
     {
         HasUnevenRows = true;
+        if (DeviceInfo.Platform == DevicePlatform.WinUI)
+        {
+            SetDynamicResource(BackgroundColorProperty, "CellDefaultBackgroundColor");
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs:66]: I think we have these changes in the other PR as well

File: Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs [hunk 3]
```diff
@@ -29,87 +32,145 @@ public ICommand SelectionChangedCommand
...
-        previousSelection = new List<object>(SelectableItems);
+        UpdateSelectedItemTappedValue(e.Item);
+    }
+
+    private void UpdateSelectedItemTappedValue(object newValue)
+    {
         switch (SelectionMode)
         {
             case ListSelectionMode.Multiple:
-                if (SelectableItems.Contains(e.Item))
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs:67]: Can you set the `SelectedItem`  to null here?

File: Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs [hunk 4]
```diff
@@ -29,87 +32,145 @@ public ICommand SelectionChangedCommand
...
                 {
-                    SelectableItems.Add(e.Item);
+                    SelectedItems.Add(newValue);
                 }
-                RaiseSelectionChangedEvent(previousSelection, selectedIndex);
+                UpdatedItems(previousSelections, SelectedItems);
                 break;
 
             case ListSelectionMode.Single:
-                SelectableItems.Clear();
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs:79]: And clear the `SelectedItems` here?

**Comment by Sofiya-kumar** [comment]
In Android, it works as expected when the default selected item is set and shows the highlighted color. In Windows, it's not working as expected since Itemsource and templetedItems is null

**Pull request closed without merging**